README.md

```diff
@@ -89,6 +89,7 @@ playground for learning and exploring.
 * **[MD045](doc/Rules.md#md045)** *no-alt-text* - Images should have alternate text (alt text)
 * **[MD046](doc/Rules.md#md046)** *code-block-style* - Code block style
 * **[MD047](doc/Rules.md#md047)** *single-trailing-newline* - Files should end with a single newline character
+* **[MD048](doc/Rules.md#md048)** *fenced-code-block-style* - Fenced code block style
```

Comment:
I think it would be compatible, since they are just extended versions of the "fenced" setting. EX: the MD46: fenced, would be MD48: consistent, and "backtick" or "tilde" just imply "fenced" + an extra check

Reply:
Renamed it to `code-fence-style`